native/wasmex/src/environment.rs

```diff
@@ -25,6 +26,50 @@ pub struct CallbackToken {
     pub return_values: Mutex<Option<(bool, Vec<WasmValue>)>>,
 }
 
+pub fn link_modules(
+    linker: &mut Linker<StoreData>,
+    store: &mut StoreOrCaller,
+    links: ListIterator,
```

Comment:
its nitpicking, so be welcome to ignore 😅  But reading this code, I think it might be easy to confuse the concept of a "linker" and linking modules through "links" here. Is this just me?
Maybe we can help a bit by clearer naming - what do you think of renaming `links` to `linked_modules`? Maybe not just here, but in the whole PR to be consistent. Might make it easier to `grep` the source code for this specific feature in the future too 🤔

Reply:
Naming is hard! I didn't like `link` that much as well. `linked_modules` makes a lot more sense.